webui/react/src/pages/TrialDetails/TrialChart.tsx

```diff
@@ -81,7 +108,7 @@ const TrialChart: React.FC<Props> = ({
       key: trialId,
       legend: { show: false },
       plugins: [ tooltipsPlugin(), trackAxis() ],
-      scales: { x: { time: false }, y: { distr: scale === Scale.Log ? 3 : 1 } },
```

Comment:
I see, so the numeric scale value was actually mapped to uPlot's scale?

Reply:
originally I was borrowing that, in the end I kept this as-is and use MetricType on the API